client/Dockerfile-5.12

```diff
@@ -150,3 +150,16 @@ RUN if [ "$BUILD_QT" != "1" ] ; then echo Download precompiled Qt. && \
       tar -xvf ${QT_TARBALL} && \
       rm ${QT_TARBALL} \
     ; fi
+
+###########################################################################
+
+# Add libqgtk
```

Comment:
I would leave this commit out, there's a good reason why this platform theme and this widgets style moved to qtstyleplugins, they're basically deprecated. qtstyleplugins is the repository where the unloved styles go to die. :-)

I think what you want is the gtk3 platform theme... but let's remove that commit and deal with this in a separated PR.

Reply:
Oki doki ;-)